### DISCUSSION TITLE
Want to buy 3 complete boards. (closed)

### DISCUSSION THREAD

**Discussion by danif86** [nliaudat/esp32_8ch_motor_shield#14] [Buy or Sell]
Need to be shipped to Norway

**Comment by tobias2611** [comment]
Hey do you sell your actuator boards?Best regards Tobi

**Reply by nliaudat** [reply]
I do not sell anything. You can order at your prefered pcb manufactory. It's open hardware

**Reply by tobias2611** [reply]
Would you give out a Gerber file for you latest version of the board?

**Reply by nliaudat** [reply]
The version with DC input ? I have not tested it under production, so I do not release gerber file actually. If you need it, you can connect to the easyeda project and generate gerber yourself.

**Comment by nethawk2** [comment]
@danif86 , are you still interested ?
I can send you 3 boards from germany, v54, rev 1.34.
Shipping to norway would cost around 25 Euros
One board did cost me around 24 Euros (assembled, shipping included).
Greetings

**Reply by danif86** [reply]
Hello, I went for heatit z-water (overpriced but a fine relee)

**Comment by roesl** [comment]
I m interested in one complete board without esp

**Reply by nethawk2** [reply]
@roesl 
I can send you one v54, rev 1.34.
After receiving the board, i would recommed, that you change resistors, like discussed here.
https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/19
Where do you live ?

**Comment by roesl** [comment]
In germany

 

Von: nethawk2 ***@***.***> 
Gesendet: Montag, 7. November 2022 23:26
An: nliaudat/esp32_8ch_motor_shield ***@***.***>
Cc: roesl ***@***.***>; Mention ***@***.***>
Betreff: Re: [nliaudat/esp32_8ch_motor_shield] Want to buy 3 complete boards. (closed) (Discussion #14)

 

@roesl <https://github.com/roesl> 
I can send you one v54, rev 1.34.
After receiving the board, i would recommed, that you change resistors, like discussed here.
#19 <https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/19> 
Where do you live ?

**Reply by nethawk2** [reply]
o.k. let's write in PM . Write to torsten at archesoft . de
